java/core/src/test/org/apache/orc/impl/TestSchemaEvolution.java

```diff
@@ -2420,50 +2423,62 @@ public void testEvolutionToTimestamp() throws Exception {
           assertEquals(expected1.replace(".1 ", " "),
               timestampToString(l1.time[current], l1.nanos[current], READER_ZONE),
               msg);
+          assertFalse(l1.isUTC(), EXPECT_LOCAL.formatted("l1"));
```

Comment:
For my understanding, is it correct, `isUTC` == `EXPECT_LOCAL`? It's a little counter-intuitive to me.